README.md

```diff
@@ -1,3 +1,127 @@
-[API Repo Linke](https://github.com/GreyMatteOr/bangwords-api) 
+[![Contributors][contributors-shield]](https://github.com/GreyMatteOr/bangwords/graphs/contributors)
+[![Forks][forks-shield]](https://github.com/GreyMatteOr/bangwords/network/members)
+[![Issues][issues-shield]](https://github.com/GreyMatteOr/bangwords/issues)
+[![Stars][stars-shield]](https://github.com/GreyMatteOr/bangwords)
 
- [API](https://bangwords-api.herokuapp.com/) 
+## BangWords (a shameless ripoff)
+
+
+<br><br>
+
+### Contributors
+
+---
+
+Matthew Lane [@GreyMatteOr](https://github.com/GreyMatteOr)
+Chris Castanuela [@Chriscastanuela](https://github.com/Chriscastanuela)
+Ian Holladay [@holladayian](https://github.com/holladayian)
+
+<br><br>
+
+### The _*sauce*_
+---
+
+`<!-- did we launch this already? -->`
+[Link to our app](https://bangwords.herokuapp.com/)
+
+[API](https://bangwords-api.herokuapp.com/) 
```

Comment:
I don't think we need to link to the server. It will just serve a `GET` request, which I think returns an `<h3>Connected!</h3>`...